tests/unit/test_smtp_relay.py

```diff
@@ -289,14 +293,21 @@ def test_configure_smtp_auth_relay_config_auth_disabled(self, call, set_flag, cl
     @mock.patch('grp.getgrnam')
     @mock.patch('os.fchown')
     @mock.patch('subprocess.call')
-    def test_configure_smtp_auth_relay_config_auth_users(self, call, fchown, getgrnam, set_flag, clear_flag):
+    def test_configure_smtp_auth_relay_config_auth_users(
+        self, call, fchown, getgrnam, set_flag, clear_flag
+    ):
         dovecot_config = os.path.join(self.tmpdir, 'dovecot.conf')
         dovecot_users = os.path.join(self.tmpdir, 'dovecot_users')
 
+        user1 = "myuser1:$1$bPb0IPiM$kmrSMZkZvICKKHXu66daQ."
+        user2 = (
+            "myuser2:$6$3rGBbaMbEiGhnGKz$KLGFv8kDTjqa3xeUgA6A1Rie1zGSf3sLT85vF1s59Yj"
+            "//F36qLB/J8rUfIIndaDtkxeb5iR3gs1uBn9fNyJDD1"
+        )
         self.mock_config.return_value[
             'smtp_auth_users'
-        ] = """myuser1:$1$bPb0IPiM$kmrSMZkZvICKKHXu66daQ.
-myuser2:$6$3rGBbaMbEiGhnGKz$KLGFv8kDTjqa3xeUgA6A1Rie1zGSf3sLT85vF1s59Yj//F36qLB/J8rUfIIndaDtkxeb5iR3gs1uBn9fNyJDD1"""
+        ] = f"""{user1}
+{user2}"""
```

Comment:
I think I would accept an ignore for this one if that's possible. I don't feel like we gain in readability with this change

Reply:
I've made some updates